cfme/tests/intelligence/test_dashboard.py

```diff
@@ -20,6 +20,14 @@
 ]
 
 
+@pytest.yield_fixture
+def widgets():
+    all_widgets = Widget.all()
+    all_widgets_ids = [widget._div_id for widget in all_widgets]
+    yield [all_widgets[0], all_widgets[1], all_widgets_ids]
```

Comment:
It's perhaps confusing to send through a list like this, perhaps a tuple, or even a named_tuple would be a nicer alternative?